org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/BuildWorkspaceResult.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.jdt.ls.core.internal;
+
+import org.eclipse.lsp4j.jsonrpc.validation.NonNull;
+import org.eclipse.xtext.xbase.lib.Pure;
+
+/**
+ * Representation of a response that server sends to client about the workspace
+ * build result
+ *
+ */
+public class BuildWorkspaceResult {
+
+	@NonNull
+	private BuildWorkspaceStatus status;
+
+	private String details;
```

Comment:
I think it will be better if details is an array of `org.eclipse.lsp4j.Diagnostic` instead of an concatenated list of compiler error strings.

Reply:
@gorkem  i'm now thinking whether to only return BuildWorkspaceStatus considering that there seems to be no place to show these diagnositics in vscode. My idea is to only show the status, what do you think about this?